### PR TITLE
Pr shared env

### PR DESCRIPTION
**Pull request by andr2000**
Use shared between domains environment to pass paths and other parameters, e.g. num OS ids

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 LICENSE = "CLOSED"
 
-IMAGES_SRC_DIR="boot/domd"
-# TODO: use Xen defined location for storing domain artifacts
-IMAGES_DST_DIR="xt/domd"
+require inc/xt_shared_env.inc
 
-FILES_${PN} += "${base_prefix}/${IMAGES_DST_DIR}"
+FILES_${PN} += "${base_prefix}${XT_DIR_ABS_ROOTFS_DOMD}"
```
Comment (lorc) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb:5]: So, it is not so absolute after all.

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb [hunk 2]
```diff
@@ -13,11 +11,11 @@ FILES_${PN} += "${base_prefix}/${IMAGES_DST_DIR}"
 # force install so if DomU kernel changes we use the latest binaries
 do_install[nostamp] = "1"
 do_install() {
-   install -d "${D}/${base_prefix}/${IMAGES_DST_DIR}"
-   find "${XT_SHARED_ROOTFS_DIR}/${IMAGES_SRC_DIR}" -iname 'Image*' -exec \
-     cp -f --no-dereference --preserve=links {} "${D}/${base_prefix}/${IMAGES_DST_DIR}" \;
+   install -d "${D}/${base_prefix}/${XT_DIR_ABS_ROOTFS_DOMD}"
```
Comment (lorc) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb]: You removed `/` at L5, but left it there.

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/domd-agl-image-copy-kernel/domd-agl-image-copy-kernel.bb [hunk 3]
```diff
@@ -1,18 +0,0 @@
-LICENSE = "CLOSED"
```
Comment (lorc) [inline recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/domd-agl-image-copy-kernel/domd-agl-image-copy-kernel.bb:1]: Commit message does not say anything about this file removal. Where did you put replacement?

File: recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend [hunk 4]
```diff
@@ -34,6 +40,8 @@ add_to_local_conf() {
     base_update_conf_value ${local_conf} SERIAL_CONSOLE "115200 hvc0"
 
     base_update_conf_value ${local_conf} PREFERRED_VERSION_xen "4.9.0+git\%"
+
+    # set folders where rootfs artifacts are stored
```
Comment (lorc) [inline recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend]: Hmm... This comment is for what exactly?

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb (hunk 1 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb:5]: bitbake.conf: export base_prefix = ""

File: recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend (hunk 4 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/dom0-image-thin-initramfs.bbappend]: done

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb (hunk 2 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/domd-install-artifacts/domd-install-artifacts.bb]: done

File: recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/domd-agl-image-copy-kernel/domd-agl-image-copy-kernel.bb (hunk 3 above)
Comment (andr2000) [reply recipes-domd/agl/files/meta-xt-prod-extra/recipes-extended/domd-agl-image-copy-kernel/domd-agl-image-copy-kernel.bb:1]: will be a separate patch

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**